core/client/fs/src/main/java/alluxio/client/file/cache/store/PageStoreOptions.java

```diff
@@ -11,11 +11,39 @@
 
 package alluxio.client.file.cache.store;
 
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.conf.PropertyKey;
+
 /**
  * Options used to instantiate a {@link alluxio.client.file.cache.PageStore}.
  */
 public abstract class PageStoreOptions {
 
+  /**
+   * Creates an instance of the {@link PageStoreOptions} based on configurations.
+   * @param conf the Alluxio configuration
+   * @return the created page store options
+   */
+  public static PageStoreOptions create(AlluxioConfiguration conf) {
```

Comment:
Since this is simple enough, I would suggest to move the body this function into factory method `PageStore.create`, inside which we can switch the creation of options based on `PropertyKey.USER_CLIENT_CACHE_STORE_TYPE`.